clang/test/3C/refarrsubscript.c

```diff
@@ -7,19 +7,19 @@
 
 int **func(int **p, int *x) {
   //CHECK_NOALL: int **func(int **p : itype(_Ptr<_Ptr<int>>), _Ptr<int> x) : itype(_Ptr<int *>) {
-  //CHECK_ALL: _Array_ptr<_Ptr<int>> func(_Array_ptr<_Ptr<int>> p, _Ptr<int> x) _Checked {
+  //CHECK_ALL: _Array_ptr<_Ptr<int>> func(_Array_ptr<_Ptr<int>> p : count(1 + 1), _Ptr<int> x) : count(1 + 1) _Checked {
```

Comment:
The return bound here is incorrect, but maybe this is acceptable since we know bounds inference gets bounds wrong some times.

Reply:
Yup. Let's make another issue for this. There are some issues with generating bounds for `&(arr[1])` kinda expressions.